pages.fa/android/cmd.md

```diff
@@ -0,0 +1,16 @@
+# cmd
+
+> مدیر سرویس اندروید.
+> اطلاعات بیشتر: <https://cs.android.com/android/platform/superproject/+/master:frameworks/native/cmds/cmd/>.
+
+- فهرست تمام سرویس های درحال اجرا:
+
+`cmd -l`
+
+- فراخوان یک سرویس:
+
+`cmd {{service}}`
+
+- فراخوان یک سرویس با مقادیر ورودی:
+
+`cmd {{service}} {{arg1 arg2 ...}}`
```

Comment:
If possible, can you translate the `{{argument1 argument2 ...}}` placeholder too?

Reply:
```suggestion
`cmd {{service}} {{استدلال 1 استدلال 2 ...}}`
```